calico_node/tests/st/bgp/test_ipip.py

```diff
@@ -90,47 +96,45 @@ def test_ipip(self, backend):
 
             # Turn on IPIP with a v1.0.2 calicoctl and check that the
             # IPIP tunnel is being used.
-            self.pool_action(host1, "replace", DEFAULT_IPV4_POOL_CIDR, True,
-                             calicoctl_version="v1.0.2")
+            self.pool_action(host1, "replace", DEFAULT_IPV4_POOL_CIDR, ipip_mode="Always",)
+                             # comment this out for now because we don't support upgrading data yet
```

Comment:
Are we tracking re-enabling this assertion somewhere?

Should we just add TODO in all the places we've commented something so that we don't lose them?

Reply:
Not yet, AFAIK, but we should!

I don't want really want to go through more cycles on _this_ PR, but I think we need an explicit follow-on process to review the changes and make sure we have all appropriate follow-on work tracked.